client/controllers/curatorEvents.coffee

```diff
@@ -4,84 +4,67 @@ CuratorSources = require '/imports/collections/curatorSources.coffee'
 Articles = require '/imports/collections/articles.coffee'
 
 Template.curatorEvents.onCreated ->
-  @eventFields = [
-    {
-      key: "expand"
-      label: ""
-      cellClass: "open-row"
-    },
-    {
-      key: 'eventName'
-      label: 'Event Name'
-      sortDirection: 1
-      hidden: false
-    },
-    {
-      key: 'creationDate'
-      label: 'Creation Date'
-      sortOrder: 0
-      sortDirection: -1
-      hidden: true
-    }
-  ]
-  @addEventMenuIsOpen = new ReactiveVar false
   @subscribe("articles", {
     url:
       $regex: "post\/" + @data._sourceId + "$"
   })
-  @associatedEventIds = new ReactiveVar([])
+  @associatedEventIdToArticle = new ReactiveVar([])
```

Comment:
Seems like the name of this reactiveVar so be plural or somehow indicate it contains multiple Ids?